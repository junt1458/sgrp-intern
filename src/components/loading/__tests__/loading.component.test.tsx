import { render } from '@testing-library/react';
import Loading from '../loading.component';

it('should render the component', () => {
  const { container } = render(<Loading message='test' />);
  expect(container).toMatchSnapshot();
});
