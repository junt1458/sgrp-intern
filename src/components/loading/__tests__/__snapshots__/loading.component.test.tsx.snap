// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the component 1`] = `
<div>
  <div
    class="my-8"
  >
    <div
      class="flex flex-wrap items-center justify-center"
    >
      <img
        alt="Loading icon"
        height="200"
        src="/loading.gif"
        width="200"
      />
      <div
        class="w-full text-center"
      >
        test
      </div>
    </div>
  </div>
</div>
`;
